backend/core/examples/chatbot/main.py

```diff
@@ -0,0 +1,63 @@
+import tempfile
+
+import chainlit as cl
+from quivr_core import Brain
+
+
+@cl.on_chat_start
+async def on_chat_start():
+    files = None
+
+    # Wait for the user to upload a file
+    while files is None:
+        files = await cl.AskFileMessage(
+            content="Please upload a text .txt file to begin!",
+            accept=["text/plain"],
+            max_size_mb=20,
+            timeout=180,
+        ).send()
+
+    file = files[0]
+
+    msg = cl.Message(content=f"Processing `{file.name}`...", disable_feedback=True)
+    await msg.send()
+
+    with open(file.path, "r", encoding="utf-8") as f:
+        text = f.read()
+
+    with tempfile.NamedTemporaryFile(
```

Comment:
tempfile are usually garbage collected as soon and the context is closed. I think we can pass that file.path directly to the brain 🤔 ? 

If this doesn't work, maybe have the logic inside the context manager